### PR TITLE
Release Consensus packages for Node 8.12.1

### PR DESCRIPTION
**Pull request by nfrisby**
This PR technically violates the ReleaseProcess.md rules. But it's trivially so: the last commit of this PR is supposed to be isolated in a separate PR.

### CODE REVIEW

File: ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Eras.hs [hunk 1]
```diff
@@ -183,12 +200,51 @@ defaultApplyShelleyBasedTx ::
...
+          totalRefScriptsSizeLimit = 100 * 1024
+
+          freeOfChargeRefScriptsBytes :: Int
+          freeOfChargeRefScriptsBytes = 50 * 1024
+
+          actualFee = tx ^. SL.bodyTxL . SL.feeTxBodyL
+          expectedFee = minFee SL.<+> refScriptsFee
+            where
+              minFee = SL.getMinFeeTx (SL.ledgerPp ledgerEnv) tx 0
+              refScriptsFee = SL.tierRefScriptFee 1.5 25600 15 refScriptsSize
```
Comment (lehins) [inline ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Eras.hs:243]: Growth factor was agreed to be `1.2`
```suggestion
              refScriptsFee = SL.tierRefScriptFee 1.2 25600 15 refScriptsSize
```

File: ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Eras.hs [hunk 2]
```diff
@@ -199,34 +255,57 @@ instance (SL.PraosCrypto c, DSignable c (Hash c EraIndependentTxBody))
...
   applyShelleyBasedTx = applyAlonzoBasedTx
 
   getConwayEraGovDict _ = Just ConwayEraGovDict
 
+  getBabbageTxDict _ = Just $ BabbageTxDict $ \a b ->
+      SL.ApplyTxError
+    $ pure
+    $ Conway.ConwayUtxowFailure
+    $ Conway.UtxoFailure
+    $ Conway.MaxTxSizeUTxO a b
```
Comment (lehins) [inline ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Eras.hs:307]: I think we should disable this patch in Conway
```suggestion
  getBabbageTxDict _ = Nothing
```

**Review by lehins — APPROVED** [review]
Looks good. Couple of changes I think still need to be made, but it would be the end of the world if we don't make them.

File: ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Eras.hs (hunk 2 above)
Comment (nfrisby) [reply ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Eras.hs]: Done, thanks.

File: ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Eras.hs (hunk 1 above)
Comment (nfrisby) [reply ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Eras.hs]: Done, thanks.

**Review by dnadales — APPROVED** [review]

**Pull request merged**